src/main/java/core/basesyntax/figures/Rectangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.figures;
+
+public class Rectangle extends Figure {
+    private int width;
+    private int height;
+
+    public Rectangle(String color, int width, int height) {
+        super(color);
+        this.width = width;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) width * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(
+                "This is a "
+                + this.getColor()
+                + " rectangle with width = " + width + " cm "
+                + " rectangle with height = " + height + " cm "
+                + " and area " + this.getArea() + " square cm"
```

Comment:
`this` are redundant

Reply:
I cant figure out how I'll fix it. As for me it's pretty well